src/init.cpp

```diff
@@ -831,6 +831,20 @@ void ThreadImport(std::vector<fs::path> vImportFiles)
     // GetMainSignals().UpdatedBlockTip(chainActive.Tip());
     pdsNotificationInterface->InitializeCurrentBlockTip();
 
+    {
+        // Get all UTXOs for each MN collateral in one go so that we can fill coin cache early
+        // and reduce further locking overhead for cs_main in other parts of code inclluding GUI
```

Comment:
nit: including

Reply:
Sigh.. I literally hate keyboard on recent macs more and more every day...